mkdocs.yml

```diff
@@ -202,3 +202,5 @@ plugins:
         404: ['https://support.google.com/accounts/answer/185833?hl=en',
               'https://support.google.com/accounts/answer/6010255?hl=en']
         429: ['*']
+        -1: ['https://www.rfc-editor.org/rfc/rfc7519',
```

Comment:
Can these be made less specific? These responses are going to be true for any pages on these domains.